File: hcode-fe/src/stdin/temp/js/source.js
function stringInArray(s) {
    const exp = new Error(`Invalid argument: ${s}`);
    try {
        const m = s.trim().match(/^\[(.+)\]$/);
        if (m) return m[1];
        throw exp;
    } catch { throw exp; }
}
function stringToInt(s) {
    const exp = new Error(`Invalid argument: ${s}`);
    try {
        const i = parseInt(s.trim());
        return !isNaN(i) ? i : exp;
    } catch { throw exp; }
}
function stringToInts(s) {
    const exp = new Error(`Invalid argument: ${s}`);
    try {
        return stringInArray(s).split(",").map(i => stringToInt(i));
    } catch { throw exp; }
}
function stringToDouble(s) {
    const exp = new Error(`Invalid argument: ${s}`);
    try {
        const i = parseFloat(s.trim());
        return !isNaN(i) ? i : exp;
    } catch { throw exp;    }
}
function stringToDoubles(s) {
    const exp = new Error(`Invalid argument: ${s}`);
    try {
        return stringInArray(s).split(",").map(i => stringToDouble(i));
    } catch { throw exp; }
}
function stringToStrings(s) {
    const exp = new Error(`Invalid argument: ${s}`);
    try {
        return stringInArray(s).split(",").map(i => i?.toString());
    } catch { throw exp; }
}
function stringToBool(s) {
    const exp = new Error(`Invalid argument: ${s}`);
    try {
        const i = JSON.parse(s.trim().toLowerCase());
        return typeof i === 'boolean' ? i : exp;
    } catch { throw exp; }
}
function stringToBools(s) {
    const exp = new Error(`Invalid argument: ${s}`);
    try {
        return stringInArray(s).split(",").map(i => stringToBool(i));
    } catch { throw exp; }
}
function splitString(inputString, delimiter) {
    return inputString.split(delimiter);
}
function printString(i) {
    console.log(i);
}
function printInt(i) {
    console.log(i.toString());
}
function printDouble(i) {
    console.log(i.toString());
}
function printBool(i) {
    console.log(i.toString());
}
function printStrings(a) {
    console.log(`[${a.join(",")}]`);
}
function printInts(a) {
    console.log(`[${a.join(",")}]`);
}
function printDoubles(a) {
    console.log(`[${a.join(",")}]`);
}
function printBools(a) {
    console.log(`[${a.join(",")}]`);
}
function main() {
    let inputString = "";
    process.stdin.on("data", (chunk) => {
        inputString += chunk.toString();
    });
    process.stdin.on("end", () => {
        const a = splitString(inputString, "|");
        console.log({output}(solve({args})));
    });
}
{userCode}
if (require.main === module) {
    main();
}

File: hcode-fe/src/stdin/temp/c/source.c
#include <stdio.h>
#include <string.h>
#include <stdlib.h>
#define MAX_TOKENS 100
#define MAX_STRING_LENGTH 2560
char* stringInArray(char* s) {
    char* exp = "Invalid argument";
    char* result = NULL;
    char* trimmed = strdup(s);
    trimmed[strcspn(trimmed, "\r\n")] = 0;

    if (trimmed[0] == '[' && trimmed[strlen(trimmed) - 1] == ']') {
        result = malloc(strlen(trimmed) - 1);
        strncpy(result, trimmed + 1, strlen(trimmed) - 2);
        result[strlen(trimmed) - 2] = '\0';
    } else {
        fprintf(stderr, "%s: %s\n", exp, s);
        exit(1);
    }

    free(trimmed);
    return result;
}
int stringToInt(char* s) {
    char* exp = "Invalid argument";
    int result;
    if (sscanf(s, "%d", &result) == 1) {
        return result;
    } else {
        fprintf(stderr, "%s: %s\n", exp, s);
        exit(1);
    }
}
int* stringToInts(char* s, int* count) {
    char* exp = "Invalid argument";
    char* str = stringInArray(s);
    char* token = strtok(str, ",");
    int* result = malloc(sizeof(int) * strlen(token));
    int i = 0;

    while (token != NULL) {
        result[i++] = stringToInt(token);
        token = strtok(NULL, ",");
    }

    *count = i;
    free(str);
    return result;
}
double stringToDouble(char* s) {
    char* exp = "Invalid argument";
    double result;
    if (sscanf(s, "%lf", &result) == 1) {
        return result;
    } else {
        fprintf(stderr, "%s: %s\n", exp, s);
        exit(1);
    }
}
double* stringToDoubles(char* s, int* count) {
    char* exp = "Invalid argument";
    char* str = stringInArray(s);
    char* token = strtok(str, ",");
    double* result = malloc(sizeof(double) * strlen(token));
    int i = 0;

    while (token != NULL) {
        result[i++] = stringToDouble(token);
        token = strtok(NULL, ",");
    }

    *count = i;
    free(str);
    return result;
}
char** stringToStrings(char* s, int* count) {
    char* exp = "Invalid argument";
    char* str = stringInArray(s);
    char* token = strtok(str, ",");
    char** result = malloc(sizeof(char*) * strlen(token));
    int i = 0;

    while (token != NULL) {
        result[i] = strdup(token);
        token = strtok(NULL, ",");
        i++;
    }

    *count = i;
    free(str);
    return result;
}
_Bool stringToBool(char* s) {
    char* exp = "Invalid argument";
    _Bool result;
    if (strcmp(s, "true") == 0 || strcmp(s, "1") == 0) {
        return 1;
    } else if (strcmp(s, "false") == 0 || strcmp(s, "0") == 0) {
        return 0;
    } else {
        fprintf(stderr, "%s: %s\n", exp, s);
        exit(1);
    }
}
_Bool* stringToBools(char* s, int* count) {
    char* exp = "Invalid argument";
    char* str = stringInArray(s);
    char* token = strtok(str, ",");
    _Bool* result = malloc(sizeof(_Bool) * strlen(token));
    int i = 0;

    while (token != NULL) {
        result[i++] = stringToBool(token);
        token = strtok(NULL, ",");
    }

    *count = i;
    free(str);
    return result;
}
void printString(char *i) {
    printf("%s", i);
}

void printInt(int i) {
    printf("%d", i);
}

void printDouble(double i) {
    printf("%f", i);
}

void printBool(int i) {
    printf("%d", i);
}
void printStrings(char **a, int size) {
    printf("[");
    for (int j = 0; j < size; ++j) {
        printf("%s", a[j]);
        if (j < size - 1) {
            printf(",");
        }
    }
    printf("]");
}
void printInts(int *a, int size) {
    printf("[");
    for (int j = 0; j < size; ++j) {
        printf("%d", a[j]);
        if (j < size - 1) {
            printf(",");
        }
    }
    printf("]");
}
void printDoubles(double *a, int size) {
    printf("[");
    for (int j = 0; j < size; ++j) {
        printf("%f", a[j]);
        if (j < size - 1) {
            printf(",");
        }
    }
    printf("]");
}
void printBools(int *a, int size) {
    printf("[");
    for (int j = 0; j < size; ++j) {
        printf("%d", a[j]);
        if (j < size - 1) {
            printf(",");
        }
    }
    printf("]");
}
char** splitString(char *inputString, char *delimiter) {
    char *token = strtok(inputString, delimiter);
    char** result = malloc(sizeof(char*) * MAX_TOKENS);
    int count = 0;
    while (token != NULL && count < MAX_TOKENS) {
        result[count] = malloc(strlen(token) + 1);
        strcpy(result[count++], token);
        token = strtok(NULL, delimiter);
    }
    result[count] = NULL;
    return result;
}
{userCode}
int main() {
    char inputString[MAX_STRING_LENGTH];
    fgets(inputString, MAX_STRING_LENGTH, stdin);
    inputString[strcspn(inputString, "\n")] = '\0';
    char **a = splitString(inputString, "|");
    {output}(solve({args}));
    return 0;
}
    
